_episodes/00-quickstart.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Quickstart Guide"
+teaching: 1
+exercises: 10
+questions:
+
+- "What is the purpose of the quickstart guide?"
+- "How do I activate and test the ESMValTool environment?"
+- "How do I configure ESMValTool?"
+- "How do I run a recipe?"
+
+objectives:
+- "Understand the purpose of the quickstart guide"
+- "Activate and test the ESMValTool environment"
+- "Configure ESMValTool"
+- "Run a recipe"
+
+keypoints:
+
+- "The purpose of the quickstart guide is to enable the user to run the 
+  ESMValTool as quickly as possible"
+- "Use the `module load` command to activate the ESMValTool environment, 
+  see the installation guide for more details, use `esmvaltool -- --help` to 
+  test the ESMValTool environment"
+- "Use `esmvaltool config get_config_user` to create the ESMValTool user 
+  configuration file"
+- "Use `esmvaltool run <recipe>.yml` to run a recipe"
+---
```

Comment:
I wonder whether we should add a note to say what version of ESMValTool this quick start guide is compatible with?
```suggestion
---
> ## Compatibility
>
> This quickstart guide is compatible with ESMValTool v2.6.0.
 {: .callout}
```